packages/nimble-components/.storybook/main.js

```diff
@@ -23,7 +23,8 @@ export const addons = [
         }
     },
     '@storybook/addon-a11y',
-    '@storybook/addon-interactions'
+    '@storybook/addon-interactions',
+    'storybook-addon-pseudo-states'
```

Comment:
This plugin seems active on every story, and it's behavior does not seem very good / useful too be always enabled.

Reply:
I've disabled the toolbar item for the addon in all stories.